File: mocksubmission/fib-cpp/fib_pass.cpp
#include <iostream>

int main() {
    int n;
    std::cin >> n;
 
    // use long long because of the fat size fib numbers get up to
    long long previous = 0;
    long long current = 1;
    for (int i = 0; i < n; i++) {
        long long next = previous + current;
        previous = current;
        current = next;
    }

    std::cout << previous << '\n';
    return 0;
}

File: mocksubmission/fib-java/FibPass.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        long previous = 0;
        long current = 1;
        for (int i = 0; i < n; i++) {
            long next = previous + current;
            previous = current;
            current = next;
        }
        System.out.println(previous);
    }
}
